.travis.yml

```diff
@@ -21,36 +21,40 @@ matrix:
     fast_finish: true
     include:
           # Minimum supported Symfony version and latest PHP version
-        - php: 7.1
-          env: DEPENDENCIES="minimum" COVERAGE=true TEST_COMMAND="composer test-ci" SYMFONY_DEPRECATIONS_HELPER="weak" SYMFONY_PHPUNIT_VERSION="5.7"
+        - php: 7.2
+          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak" SYMFONY_PHPUNIT_VERSION="5.7"
 
           # Test the latest stable release
+        - php: 5.5
+          env: SYMFONY_PHPUNIT_VERSION="5.7"
         - php: 5.6
           env: SYMFONY_PHPUNIT_VERSION="5.7"
         - php: 7.0
         - php: 7.1
         - php: 7.2
+          env: COVERAGE=true TEST_COMMAND="composer test-ci"
 
           # Test LTS versions
-        - php: 5.5
-          env: DEPENDENCIES="symfony/lts:^2"
-        - php: 5.5
-          env: DEPENDENCIES="symfony/lts:^3"
+        - php: 7.2
+          env: DEPENDENCIES="dunglas/symfony-lock:^2"
+        - php: 7.2
+          env: DEPENDENCIES="dunglas/symfony-lock:^3"
+        - php: 7.2
+          env: DEPENDENCIES="dunglas/symfony-lock:^4" STABILITY="rc"
 
           # Latest commit to master
-        - php: 7.1
-          env: DEPENDENCIES="dev"
+        - php: 7.2
+          env: STABILITY="dev"
 
     allow_failures:
           # dev-master is allowed to fail.
-        - php: 7.1
-          env: DEPENDENCIES="dev"
+        - php: 7.2
```

Comment:
according to the symfony docs PR, you can drop this and just keep the bit about env to allow failures

Reply:
You mean:
```yaml
allow_failures:
    env: DEPENDENCIES="dev"
```
?

Reply:
yes. see https://github.com/symfony/symfony-docs/pull/8701/files#diff-9d0d9156660f746e55727b52fcbf918dR214 - it will be good to see here if that works as expected ;-)

Reply:
We will not be able to know this with this PR as its 100% green ;)

Reply:
we can see it: when you click on the build, you see a travis page where the allowed to fail build are separated, even if they succeed.

the point of omitting the php version is that we don't care about the version and when php 7.3 comes out we would forget to adjust this line.